sympy/printing/rust.py

```diff
@@ -526,10 +553,17 @@ def _print_ITE(self, expr):
         return self._print(expr.rewrite(Piecewise, deep=False))
 
     def _print_MatrixBase(self, A):
-        if A.cols == 1:
-            return "[%s]" % ", ".join(self._print(a) for a in A)
-        else:
-            raise ValueError("Full Matrix Support in Rust need Crates (https://crates.io/keywords/matrix).")
+        # Print all matrices like matrix, i.e. not vector.
+        # Rust is a staticly typed compiled language. Unlike python, [type; N] in rust is a fixed size type that is suitable to be used for matrix.
```

Comment:
I'm not sure I understand this. Is this changing the type?

Long comment lines should be wrapped to 78 characters.

Reply:
To be clear though is this going to break anyone who is currently using the Rust printer to print column matrices.